bindings/py/requirements.txt

```diff
@@ -1,6 +0,0 @@
-#See http://www.pip-installer.org/en/latest/requirements.html for details
-numpy==1.12.1
```

Comment:
is this file moved somewhere else? Even if we remove some deps, we still use it to install others? (numpy,..) 
CI files (.travis.yaml, circle.yml, appveyor.yml) depend install from it.

Reply:
Not sure we need this folder anymore but if CI scripts insist I can bring it back.